org.gnome.ColorProfileViewer.json

```diff
@@ -0,0 +1,127 @@
+{
+  "app-id": "org.gnome.ColorProfileViewer",
+  "sdk": "org.gnome.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "command": "gcm-viewer",
+  "branch": "stable",
```

Comment:
this bit is redundant on flathub and fwiw I find it annoying to have specified for local dev (where I generally want my version to be 'master'